src/vstt/trial.py

```diff
@@ -33,6 +33,7 @@ def default_trial() -> Trial:
         "target_indices": "0 1 2 3 4 5 6 7",
         "add_central_target": True,
         "hide_target_when_reached": True,
+        "turn_target_to_green_when_reached": True,
```

Comment:
When we add a new option, the default value should be chosen to maintain the previous behaviour for backward compatibility with existing experiments
```suggestion
        "turn_target_to_green_when_reached": False,
```